apps/client/src/app/structure-definition/structure-definition.component.ts

```diff
@@ -300,6 +302,22 @@ export class StructureDefinitionComponent extends BaseComponent implements OnIni
       }, (err) => {
         this.message = getErrorString(err);
       });
+
+    const implementationGuideId = this.route.snapshot.paramMap.get('implementationGuideId');
+    const implementationGuide = await this.implementationGuideService.getImplementationGuide(implementationGuideId).toPromise();
+    const resources = (<ImplementationGuide> implementationGuide).definition.resource;
+
+
+    const index = resources.findIndex(resource => {
+      return resource.reference.reference.indexOf(this.structureDefinition.id) > 0;
+    });
+    if(index === -1) throw new Error(`Couldn't find structure definition in resources`);
```

Comment:
Don't throw an error here. Just wrap the next block with
`if (index >= 0) { ... }`